app/routes/team/index.js

```diff
@@ -35,6 +35,12 @@ export default AuthenticatedRoute.extend(styleBody, CurrentUserSettings, Paginat
                 });
             }
 
+            // we need to load the roles into ember cache
+            // invites return role_id only and we do not offer a /role/:id endpoint
+            modelPromises.roles = this.get('store').query('role', {permissions: 'assign'}).then((roles) => {
```

Comment:
Is the `{permissions: 'assign'}` needed here? It would be preferable to load all roles so that the role of team members can be displayed even if the current user doesn't have permission to assign those roles to new users.